README.md

```diff
@@ -32,24 +32,47 @@ oc new-project ocpdoom
 
 ```bash
 oc create serviceaccount doomguy -n ocpdoom
-oc create clusterrole monster-control --verb=get,list,watch,kill --resource=pods
-oc adm policy add-cluster-role-to-user monster-control -z doomguy -n ocpdoom
+oc create role monster-control --verb=get,list,watch,delete --resource=pods -n monsters
+```
+
+```
+# need to create service account rolebinding via YAML
+# oc adm policy add-role-to-user monster-control --role-namespace monsters doomguy -n monsters
+
+cat << YAML | oc apply -f -

```

Comment:
Please trim this out to keep the oc command theme.

Reply:
So remove the commented lines?

This was the easiest way I knew how to do it using EOL. Do you have another way?

Reply:
`oc adm policy add-role-to-user monster-control --role-namespace monsters doomguy -n monsters` does not work. You aren't able to set the namespace for the service account that exists in another namespace.

The YAML is the only way I know to make this work. `oc adm policy` does not allow this.

Reply:
This should do it:

`oc create rolebinding --role monster-control --serviceaccount ocpdoom:doomguy -n monsters monster-control`

Reply:
Let me test that. I wonder I'll make note of what version of `oc` I was using, because I have inconsistent results with different versions.

Reply:
Yep, that worked. Thanks.